<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemUpdate</class>
 <widget class="QDialog" name="ItemUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>362</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelAppName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #4382EC;
	font: 900 11pt &quot;Roboto&quot;;
	border-radius: 10px;
	text-align: center;
	background-color: #E2EEFF;
}
	</string>
   </property>
   <property name="text">
    <string>  Super Market Management System  </string>
   </property>
  </widget>
  <widget class="QFrame" name="frameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame 
{
background-color: #E2EEFF;
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="itemCode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color : #BBD6FD;
color: #4382EC;
border: 2px solid #4382EC;
border-radius: 10px;
font-weight: bold;
}

QLineEdit:hover
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
border-radius: 10px;
font-weight: bolder;
}

QLineEdit:focus
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
border-radius: 10px;
color: #4382EC;
font-weight: italic;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter Item code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemPrice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color : #BBD6FD;
color: #4382EC;
border: 2px solid #4382EC;
border-radius: 10px;
font-weight: bold;
}

QLineEdit:hover
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
border-radius: 10px;
font-weight: bolder;
}

QLineEdit:focus
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
border-radius: 10px;
color: #4382EC;
font-weight: italic;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter Item Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemQuantity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color : #BBD6FD;
color: #4382EC;
border: 2px solid #4382EC;
border-radius: 10px;
font-weight: bold;
}

QLineEdit:hover
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
border-radius: 10px;
font-weight: bolder;
}

QLineEdit:focus
{
background-color : #BBD6FD;
border: 3px solid #4382EC;
border-radius: 10px;
color: #4382EC;
font-weight: italic;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Enter Item Quantity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
image:url(&quot;:/images/Clear.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
image:url(&quot;:/images/Save.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
	padding: 2px;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 238, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image:url(&quot;:/images/Back.svg&quot;);
background-color: #E2EEFF;
color: white;
border-radius: 10px;
text-align: center;
}

QPushButton:hover {
	color: #4382EC;
	font-weight: bold;
	background-color: #E2EEFF;
	border-radius: 10px;
	border: 2px solid #4382EC;
}

QPushButton:pressed {
color: #E2EEFF;
border: 4px solid #BBD6FD;
}
	</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
